lms/djangoapps/verify_student/models.py

```diff
@@ -1186,3 +1188,62 @@ def check_user_skipped_reverification_exists(cls, user, course_id):
             Boolean
         """
         return cls.objects.filter(user=user, course_id=course_id).exists()
+
+
+class CreditCourse(models.Model):
+    """Model for tracking the credit course."""
+
+    course_key = CourseKeyField(max_length=255, db_index=True, unique=True)
+    enabled = models.BooleanField(default=False)
+
+
+class CreditProvider(TimeStampedModel):
+    """This model represents an institution that can grant credit for a course.
+    each provider is identified by unique id e.g ASU
+    """
+    provider_id = models.CharField(max_length=255, db_index=True, unique=True)
+    display_name = models.CharField(max_length=255)
+
+
+class CreditRequirement(TimeStampedModel):
+    """This model represents a credit requirement.
+
+    Each requirement is uniquely identified by a `namespace` and a `name`. CreditRequirements
+    also include a `configuration` dictionary, the format of which varies by the type of requirement.
+    The configuration dictionary provides additional information clients may need to determine
+    whether a user has satisfied the requirement.
+    """
+
+    course = models.ForeignKey(CreditCourse, related_name="credit_requirement_course")
```

Comment:
The `related_name` attribute specifies the name of the reverse relation from the `CreditCourse` model back to the `CreditRequirement` model. That being the case, this related name is not accurate. It should be `'credit_requirements'`.